run_uh.py

```diff
@@ -343,7 +339,7 @@ def setup_fife():
 	log_sys_info()
 	if not find_fife():
 		#TODO useful error message anyone?
```

Comment:
I'm wondering, can we drop this TODO then? Or what makes a useful error message here? Perhaps the list of locations we searched for?